src/analytics/serverless.yml

```diff
@@ -28,8 +28,8 @@ package:
 functions:
   product:
     name: datalake-analytics-api-${self:provider.stage}-store
-    handler: analytics/index.handler
+    handler: analytics/store-analytics.handler
     package:
       include:
-        - ../analytics/index.js
+        - ../analytics/store-analytics.js
```

Comment:
It would be better to add dependencies as needed otherwise you might have a bigger bundle and it might take longer to init. Besides that if the bundle is small you can modify the code online from Lambda console.
It is a minor, change it if you want.

Reply:
We had already solved it by having one package.json per lambda, so it's ok as it is.

Reply:
How to do it?

Reply:
Don't need to, my bad. Didn't get that you were using one `package.json` per folder